### DISCUSSION TITLE
Sharing features (codes, notebooks, configs) between Kedro projects.

### DISCUSSION THREAD

**Discussion by vinialbert** [kedro-org/kedro#1852] [Idea]
Hello, everyone.

I'm trying to understand how is the best way to test/reuse a part from a Kedro projetct in another Kedro project. My point is, is there any way to share configs, catalogs, pipelines, codes, notebooks, between two or more Kedro projects, without too much manual copy-paste?

**Comment by foxale** [comment]
We aim to use [micro-packaging](https://kedro.readthedocs.io/en/stable/nodes_and_pipelines/micro_packaging.html) in near future for sharing similar transformations between projects and we already develop our own custom [starters](https://kedro.readthedocs.io/en/stable/get_started/starters.html) serving as project templates.
